keras/preprocessing/timeseries.py

```diff
@@ -61,7 +61,7 @@ def timeseries_dataset_from_array(
       `data[i], data[i + r], ... data[i + sequence_length]`
       are used for create a sample sequence.
     batch_size: Number of timeseries samples in each batch
-      (except maybe the last one).
+      (except maybe the last one). If `None`, it doesn't make batches.
```

Comment:
", the data will not be batched (the dataset will yield individual samples)."

Reply:
I changed as you suggested, Thank you!